drivers/timer/nrf_rtc_timer.c

```diff
@@ -11,7 +11,6 @@
 #include <arch/arm/cortex_m/cmsis.h>
 #include <sys_clock.h>
 
-extern void youve_print(void);
```

Comment:
Exclude this file  change from this commit (split into another commit/PR with appropriate commit message)

Reply:
Done. I will create a PR shortly with these clean up changes.